docker/gaffer-jhub-options-server/volumes.js

```diff
@@ -0,0 +1,77 @@
+
+const K8sWatcher = require('./k8s-watcher')
```

Comment:
Might be worth extracting some of this to Gaffer tools and adding unit tests around it. Not really in scope for this PR but might be something worth considering, should this be used in production.